frontend/src/layouts/PrivateRoute.test.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { describe, it, expect } from "vitest";
+import { render } from "@testing-library/react";
+
+import PrivateRoute from "../layouts/PrivateRoute";
+
+describe("#PrivateRoute", () => {
+  it("should return children components when user is authenticated", () => {
+    expect(true).toBe(true);
+    // const children = <div>Children Components</div>;
```

Comment:
Leaving remarks for missing test coverage!

Reply:
`<PrivateRoute></PrivateRoute>` not present yet at the `app.jsx` yet!